src/stream/stream_backend.rs

```diff
@@ -398,6 +420,18 @@ fn create_redirect_stream(
 fn create_webrtc_turn_stream(
     video_and_stream_information: &VideoAndStreamInformation,
 ) -> Result<StreamType, SimpleError> {
+    if !is_gstreamer_plugin_available(WEBRTCSINK_PLUGIN_NAME, WEBRTCSINK_VERSION) {
```

Comment:
we should add on the same place as the webrtc file a test like:
```
`is_webrtcsink_available() -> bool`
```
Where it'll print the error message.